framework/src/tags/i18n.tag

```diff
@@ -6,10 +6,8 @@ if (_keys) {
 }
 js_messages=new com.google.gson.Gson().toJson(ymessages);
 }%
-#{if !_noScriptTag}
 <script type="text/javascript">
-#{/if}
-var i18nMessages = ${js_messages};
+var i18nMessages = ${js_messages.raw()};
```

Comment:
Isn't this line adding a XSS opportunity for attackers where at least one of the translations include something which is coming from the user? There's almost always at least one of that kind of translations in every bigger than average project.